invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/components/DocumentListEntry/DocumentListEntry.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Grid,
+  Header,
+  Icon,
+  Image,
+  Item,
+  Label,
+  List,
+} from 'semantic-ui-react';
+import { getCover } from '../../../frontsite/config';
+import { BackOfficeRoutes } from '../../../../routes/urls';
+import _isEmpty from 'lodash/isEmpty';
+
+const getRestrictions = meta => {
+  if (_isEmpty(meta._access)) return null;
+  return {
+    corner: 'left',
+    color: 'red',
+    icon: 'lock',
+    title: 'This document is Restricted',
```

Comment:
To week showing the word `Document` in the ui?

Reply:
Couldn't agree more, what about `This ${meta.document_type.toLowerCase()} is restricted`

Reply:
Good idea 👍 Please keep it in mind whenever you see `document` around